mappings/net/minecraft/client/font/TextRenderer.mapping

```diff
@@ -75,7 +75,7 @@ CLASS cvk net/minecraft/client/font/TextRenderer
 	METHOD a setFonts (Ljava/util/List;)V
 		ARG 1 fonts
 	METHOD a setRightToLeft (Z)V
-	METHOD b getStringWidth (Ljava/lang/String;)I
+	METHOD b getWidth (Ljava/lang/String;)I
```

Comment:
`getStringWidth` or `getWidthOf` are more clear, imo. Any reason for changing this?

Reply:
yeah, some name in one of the deobf realms things. Doesn't need to change tho